docs/operations/vmctl.md

```diff
@@ -57,34 +52,30 @@ on, KubeVirt ensures the same behavior for VirtualMachines.
 The new VirtualMachine will be set to the running state regardless of
 the prototype VM’s state.
 
-Implementation
---------------
+## Implementation
 
 Vmctl is implemented as a go binary, deployed in a container, that takes
 the following parameters:
 
--   `namespace`: The namespace to create the derived VirtualMachine in.
+- `namespace`: The namespace to create the derived VirtualMachine in.
     The default namespace is `default`.
-
--   `proto-namespace`: The namespace the prototype VM is in. This
+- `proto-namespace`: The namespace the prototype VM is in. This
     defaults to the value used for `namespace` if omitted.
-
--   `hostname-override`: Mainly for testing–in order to make it possible
+- `hostname-override`: Mainly for testing–in order to make it possible
     to run vmctl outside of a pod.
 
 vmctl has a single positional argument:
 
--   prototype VM name
+- prototype VM name
 
 When the vmctl container is deployed, it will locate the requested
 prototype VM, clone it, and watch wait. When the vmctl pod is deleted,
 vmctl will clean up the derived VirtualMachine. Consequently it is
 inadvisable to use a 0 length grace period for shutting down the pod.
 
-Services
---------
+## Services
 
-One note worth stressing is that from Kubernete’s perspective the vmctl
+One note worth stressing is that from Kubernetes's perspective the vmctl
```

Comment:
Not a native english speaker here, but isn't this supposed to be written: `Kubernetes'` (i.e. not an s before and after the tilde)

Reply:
https://english.stackexchange.com/questions/35654/joness-or-jones

Another example is `I went to the Jones's house`

There is no absolute Oxford English way.  Kubernetes is the project name and singular so the root must be Kubernetes.  That part is easy.  Kubernetes has possession so I add a `'s` and move on.  The plural possessive rule used to be `Jones'`.  I've had many English teachers over the years bounce back and forth.  Everyone is tired of it.  `'s` and continue with life.